contrib/msggen/msggen/model.py

```diff
@@ -204,15 +204,14 @@ def from_js(cls, js, path):
             props = var.get('properties', None)
```

Comment:
Notice how we could collapse the various cases here by using `.get("properties", {})` which means we either get a dict with or without values, the case of an empty dict means we didn't have the property set, or it was empty. In neither cases we need to do much.

The `if` below can be `if properties != {}:` which is as simple as it gets 😊

Reply:
Almost but cases like `wait` exists where:
```
if
X
then
"status": {
                  "type": "string",
                  "enum": [
                    "unpaid",
                    "paid",
                    "expired"
                  ],
if
Y
then
"status": {
                  "type": "string",
                  "enum": [
                    "offered",
                    "settled",
                    "failed",
                    "local_failed"
                  ],
if
Z
then
"status": {
                  "type": "string",
                  "enum": [
                    "pending",
                    "failed",
                    "complete"
                  ],
```
And they are not getting merged like this

Reply:
Ah I see :-) Thanks for the clarification